wsgi/addons.rst

```diff
@@ -0,0 +1,43 @@
+===========================
+Useful Addons and Utilities
+===========================
+
+Sentry
+======
+
+.. sidebar:: Build now
+
+    Run buildout for this section:
+
+    ..  code-block:: bash
+
+        buildout -c sentry.cfg
+
+`Sentry <https://sentry.io>`_ is a widely used open source monitoring and error tracking tool.
+The existing Zope integration in the legacy `ravem <https://pypi.python.org/project/raven>`_ package does not work in the WSGI pipeline.
+This is because it's component definition depends on logging components that are only available through zopeschema.xml but not wsgischema.xml.
```

Comment:
```suggestion
This is because its component definition depends on logging components that are only available through ``zopeschema.xml`` but not ``wsgischema.xml``.
```